front/src/infoboxes/Infoboxes.module.scss

```
@@ -1,94 +1,96 @@
-.exclamation {
-  position: absolute;
-  bottom: 1px;
-  height: 20px;
-  width: 20px;
-  margin-right: 16px;
-  margin-bottom: 0px;
-  clear: both;
-  min-height: 22px;
-  min-width: 22px;
-}
-
-.info-flex {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  margin-bottom: 32px;
-}
-
-.info-item {
-  display: flex;
-  min-width: 320px;
-  width: 30vw;
-  height: 38px;
-  justify-content: left;
-}
-
-.box {
-  float: left;
-  height: 20px;
-  width: 20px;
-  margin-right: 8px;
-  margin-left: 0px;
-  margin-bottom: 0px;
-  border: 1px solid black;
-  clear: both;
-  min-height: 20px;
-  min-width: 20px;
-}
-
-#open-info {
-  display: flex;
-  background: #658f60;
-}
-
-#range-officer-info {
-  display: flex;
-  background: #b2d9ad;
-}
-
-#closed-info {
-  display: flex;
-  background: #c97b7b;
-}
-
-#closed-info2 {
-  display: flex;
-  background: #95d5db;
-}
-
-#no-info {
-  display: flex;
-  background: #f2f0eb;
-}
-
-#onway-info {
-  display: flex;
-  background: #f2c66d;
-}
-
-.no-flex {
-  display: inline-block;
-}
-
-.info-text {
-  display: inline;
-  overflow: auto;
-  margin-bottom: 0;
-}
-
-.relative-text {
-  position: absolute;
-  bottom: 0;
-  left: 30px;
-}
-
-.info-item-img {
-  position: relative;
-  display: flex;
-  min-width: 320px;
-  width: 30vw;
-  height: 38px;
-  justify-content: left;
-}
+@import '../colors.module.scss';
+
+.exclamation {
+  position: absolute;

```

This class is also always used w/ .noFlex
```suggestion
.exclamation {
  @extend .noFlex;
  position: absolute;

```